plugins/dataTypes/Boolean/schema.json

```diff
@@ -0,0 +1,11 @@
+{
+	"title": “Boolean”,
```

Comment:
I think the double quotes here may be invalid... they're showing up as specially encoded ones: maybe just copy + paste the other double quotes used in the file and overwrite them.